diff --git a/Makefile b/Makefile
index 1fe97dc..bc81c35 100644
--- a/Makefile
+++ b/Makefile
@@ -27,7 +27,7 @@ runner-ubi-fips:
 
 .PHONY: helper-ubuntu
 helper-ubuntu:
-	cd dockerfiles/runner-helper && docker buildx bake ubuntu --set $(PLATFORM) --set $(OUTPUT) --builder docker-builder
+	cd dockerfiles/runner-helper && docker buildx bake ubuntu --set $(PLATFORM) --set $(OUTPUT)
 
 .PHONY: helper-alpine-latest
 helper-alpine-latest:
diff --git a/dockerfiles/runner-helper/Dockerfile.ubuntu b/dockerfiles/runner-helper/Dockerfile.ubuntu
index 25deb93..1d84e88 100644
--- a/dockerfiles/runner-helper/Dockerfile.ubuntu
+++ b/dockerfiles/runner-helper/Dockerfile.ubuntu
@@ -1,4 +1,4 @@
-FROM ubuntu
+FROM lcr.loongnix.cn/debian:14
 
 # gitlab-runner-helper will try to resolve `sh` from the path. We ensure the PATH is populated by default, as some container runtimes do no longer set a default (e.g. containerd v1.2.8)
 ENV PATH="${PATH:-/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin}"
